src/main/java/modtrekt/logic/parser/AddCommandParser.java

```diff
@@ -32,16 +32,14 @@ public Command parse(String args) throws ParseException {
 
         if (arePrefixesPresent(argMultimap, CliSyntax.PREFIX_MODULE, CliSyntax.PREFIX_MOD_NAME,
                 CliSyntax.PREFIX_MOD_CODE, CliSyntax.PREFIX_MOD_CREDIT)) {
+
             ModName name = ParserUtil.parseName(argMultimap.getValue(CliSyntax.PREFIX_MOD_NAME).get());
             ModCode code = ParserUtil.parseCode(argMultimap.getValue(CliSyntax.PREFIX_MOD_CODE).get());
             ModCredit credit = ParserUtil.parseCredit(argMultimap.getValue(CliSyntax.PREFIX_MOD_CREDIT).get());
 
             Module module = new Module(code, name, credit);
 
             return new AddCommand(module);
-        } else if (arePrefixesPresent(argMultimap, CliSyntax.PREFIX_MODULE, CliSyntax.PREFIX_MOD_NAME)
```

Comment:
Perhaps we should leave this in first, this enforces that the user has to use a specific command, so adding extra prefixes will not allow them to add the module